getting-started.md

```diff
@@ -57,12 +100,11 @@ The snapshot generation engine is deployed on [Google Kubernetes Engine](https:/
 
 All source code is [open source](https://github.com/midl-dev/tezos-snapshot-generator) so anyone can deploy a separate snapshot generator setup in the cloud.
 
+### Brought to you by Oxhead Alpha
 
-### Brought to you by MIDL.dev
-
-We are [MIDL.dev](https://midl.dev), a blockchain infrastructure company.
+We are [Oxhead Alpha](https://www.oxheadalpha.com/), a blockchain infrastructure company.
 
-We maintain the [Tezos Suite](https://midl.dev/tezos-suite/) of open-source software projects to deploy secure bakers and nodes.
+We maintain the [Tezos Suite](https://medium.com/the-aleph) of open-source software projects to deploy secure bakers and nodes.
```

Comment:
"We maintain a series of open source infrastructure projects to further the tezos ecosystem" and link to the github page instead